capstone-app/src/app/modules/photos/components/image-container/image-container.component.ts

```diff
@@ -1,52 +1,67 @@
 import { Photo } from 'src/app/modules/photos/photo-template';
 import { PhotoFetcher } from 'src/app/modules/photos/services/photo-fetcher.service';
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input, OnChanges, Output, EventEmitter } from '@angular/core';
 
 @Component({
   selector: 'app-image-container',
   templateUrl: './image-container.component.html',
   styleUrls: ['./image-container.component.scss'],
 })
-export class ImageContainerComponent implements OnInit {
-  photos: Photo[] = [];
+export class ImageContainerComponent implements OnInit, OnChanges  {
+  displayPhotos: Photo[] = [];
+  originalPhotos: Photo[] = [];
   query: string;
-  limit: number;
   errorMessage: string;
+  @Input() limit: number;
+  @Output() limitChange = new EventEmitter<number>();
```

Comment:
Might be a good idea to add some jsdoc for these inputs/outputs
Basic examples:
https://jsdoc.app/about-getting-started.html